website/static/js/profile.js

```diff
@@ -536,7 +537,15 @@ var SocialViewModel = function(urls, modes) {
         ko.observable().extend({trimmed: true, cleanup: cleanByRule(socialRules.github)}),
         self, 'github', 'https://github.com/'
     );
-
+    self.academia1 = extendLink(
+        ko.observable().extend({trimmed: true, cleanup: cleanByRule(socialRules.academia)}),
+        self, 'academia1', 'https://'
+    );
+    self.academia2 = extendLink(
```

Comment:
It would be better to give these two more descriptive names, such as `academiaInstitution` and `academiaProfileID`